pkg/logger/logger.go

```diff
@@ -0,0 +1,83 @@
+// Package logger
+//nolint: logrlint
```

Comment:
So there is this weird checker that doesn't like me passing in the keysAndValues as they are because potentionally, they could be an odd number. 🤷